codebase/superdarn/src.lib/tk/rpos.1.7/src/cnvtcoord.c

```diff
@@ -5,36 +5,37 @@
 */
 
 /*
- LICENSE AND DISCLAIMER
+ LICENSE AND DISCLAIM
 
- Copyright (c) 2012 The Johns Hopkins University/Applied Physics Laboratory
+ Copyright (c) 2012 The Johns Hopkins Univsity/Applied Physics Laboratory
 
  This file is part of the Radar Software Toolkit (RST).
 
  RST is free software: you can redistribute it and/or modify
- it under the terms of the GNU Lesser General Public License as published by
- the Free Software Foundation, either version 3 of the License, or
- any later version.
+ it und the terms of the GNU Lesser General Public License as published by
+ the Free Software Foundation, eith version 3 of the License, or
+ any lat version.
 
  RST is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- GNU Lesser General Public License for more details.
+ MCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Less General Public License for more details.
 
- You should have received a copy of the GNU Lesser General Public License
+ You should have received a copy of the GNU Less General Public License
  along with RST.  If not, see <http://www.gnu.org/licenses/>.
 */
```

Comment:
Looks like some typos have been introduced here - what's going on?

Reply:
Ok, I see there are many instances of this. Perhaps something related to the find & replace mishap?

Reply:
@mts299 I think there is still a problem here. Lots of "er" letter combinations have been erroneously removed from the comments. Can you please check the entire `cnvtcoord.c` file?

Reply:
Yea sorry meant to respond to this yesterday. However, my furnace blew a fuse so the repair guy came just as I was about to commit the code. Check it out now.

Reply:
@mts299 There were still a few problems remaining in this file so I've added change suggestions to fix them. Once you commit them I think we're good to go.

Reply:
Sweet! sorry, learned my lesson on find and replace :p